src/System.Runtime.Extensions/tests/System/EnvironmentTests.cs

```diff
@@ -175,10 +175,18 @@ public void Version_MatchesFixedVersion()
         }
 
         [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.Uap)] // Throws InvalidOperationException in Uap as NtQuerySystemInformation Pinvoke is not available
         public void WorkingSet_Valid()
         {
             Assert.True(Environment.WorkingSet > 0, "Expected positive WorkingSet value");
         }
+        
+        [Fact]
+        [SkipOnTargetFramework(~TargetFrameworkMonikers.Uap)] 
+        public void WorkingSet_Valid_Uap()
+        {
+            Assert.Throws<InvalidOperationException>(() => Environment.WorkingSet);
```

Comment:
Yeah, I know that's why it's doing IOE.  I guess I mean: the user did nothing wrong and asked for something completely valid. In the next version we will likely make it work. So maybe PNSE makes more sense. But I don't feel strongly. Up to you.